tests/integration/security/authorization_test.go

```diff
@@ -476,43 +476,129 @@ func TestAuthorization_IngressGateway(t *testing.T) {
 				Name     string
 				Host     string
 				Path     string
+				IP       string
 				WantCode int
 			}{
 				{
 					Name:     "allow www.company.com",
 					Host:     "www.company.com",
 					Path:     "/",
+					IP:       "172.16.0.1",
 					WantCode: 200,
 				},
 				{
 					Name:     "deny www.company.com/private",
 					Host:     "www.company.com",
 					Path:     "/private",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
 				{
 					Name:     "allow www.company.com/public",
 					Host:     "www.company.com",
 					Path:     "/public",
+					IP:       "172.16.0.1",
 					WantCode: 200,
 				},
 				{
 					Name:     "deny internal.company.com",
 					Host:     "internal.company.com",
 					Path:     "/",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
 				{
 					Name:     "deny internal.company.com/private",
 					Host:     "internal.company.com",
 					Path:     "/private",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
+				{
+					Name:     "allow remoteipblocks.company.com",
+					Host:     "www.company.com",
```

Comment:
The host is setting to `www.company.com` which means it will not match any of the rules in the policy, this means we won't know for sure if the IP part is matched or not, we should change it to `remoteipblocks.company.com`.

Reply:
I removed that test as I don't think it is needed.